realtime_crowd_insights/realtime_crowd_insights/ContentView.swift

```diff
@@ -14,9 +14,9 @@ struct ContentView: View {
     @State private var image: Image? = nil
  
     var body: some View {
-        TabView {
-            Text(String("History Tab"))
-                .font(.title)
+        TabView(selection: $selection){
```

Comment:
Tengo duda de esto porque así estaba originalmente cuando yo le di push (porque así lo puso Swift al crear una tabbed app), luego Mau lo cambio porque a él le marcaba error, y luego se vuelve a cambiar a lo original. No sé si a alguien le marque error esto pero podemos por mientras dejarlo así con estos cambios, y si a alguien le marca error ver cómo lo solucionamos para no andar back and forth con esto.